pom.xml

```diff
@@ -9,8 +9,8 @@
     <version>1.0-SNAPSHOT</version>
 
     <properties>
-        <maven.compiler.source>18</maven.compiler.source>
-        <maven.compiler.target>18</maven.compiler.target>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
I don't know.Is it problem when we have different version? Mine is 18

Reply:
Mine is 11. I think it is not going to support 18. Jilili, Vusal and I had this problem yesterday!